examples/cra-kitchen-sink/src/stories/type-safe-select.stories.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
Should move this into the official example for addon knobs, the cra example should be a lean and simple showcase our compatibility with CRA, not demo every possible addon/options. That's what the official example is for 😄